### DISCUSSION TITLE
Possible to limit indexing?

### DISCUSSION THREAD

**Discussion by mattdale77** [blockscout/blockscout#9456] [Q&A]
Hi,

I've been asked to set up a blockchain explorer primarily to query transactions by address. Naturally I ended up here. However going back to genesis seems overkill when all the transactions will be in the future from now. Is there a configuration or way to set an origin date or block to stop indexing at?

Apart from the computational resources the 11TB needed to index mainnet would be problematic.

I'm open to other solutions if what I am asking is not possible with Blockscout.

**Comment by sl1depengwyn** [comment]
You can do it via `BLOCK_RANGES` env variable, more info [here](https://arc.net/l/quote/qsmogqom)

**Reply by mattdale77** [reply]
Ah yes, this looks perfect. I just need to work out a suitable start block and then run till latest and I think that'll do what is required.

Thank you
